resources/views/settings/api.blade.php

```diff
@@ -6,32 +6,156 @@
 ])
 
 @section('content')
-    <x-hero>
-        <x-slot name="title">{{ __('settings.api.title') }}</x-slot>
-        <x-slot name="subtitle">{{ __('settings.api.helper') }}</x-slot>
-        <x-slot name="link">
-            <a href="{{ route('larecipe.index') }}" class="">
-                <x-icon class="link" />
-                {{ __('front.features.api.link') }}
-            </a>
-        </x-slot>
-    </x-hero>
+    @include('partials.errors')
+    <div class="container mx-auto p-4 max-w-4xl">
+        <x-hero>
+            <x-slot name="title">{{ __('settings.api.title') }}</x-slot>
+            <x-slot name="subtitle">{{ __('settings.api.helper') }}</x-slot>
+            <x-slot name="link">
+                <a href="{{ route('larecipe.index') }}" class="">
+                    <x-icon class="link" />
+                    {{ __('front.features.api.link') }}
+                </a>
+            </x-slot>
+        </x-hero>
 
-    <x-grid type="1/1" id="api">
-        @include('partials.errors')
+        <div class="flex justify-between items-center mb-6">
+            <span class="text-lg">
+                Personal Access Tokens
+            </span>
+            <a href="{{ route('settings.api.create') }}" class="btn2 btn-primary btn-outline btn-sm"
+                data-toggle="dialog-ajax" data-target="primary-dialog" data-url="{{ route('settings.api.create') }}">
+                <x-icon class="fa-solid fa-plus" />
+                <span class="hidden lg:inline">Create New Token</span>
+            </a>
+        </div>
 
-        <div class="flex flex-col xl:grid xl:grid-cols-2 gap-5">
-            <passport-personal-access-tokens></passport-personal-access-tokens>
-            <passport-authorized-clients></passport-authorized-clients>
+        @if (session('new_token'))
+            <div class="bg-green-100 text-green-900 rounded p-4 mb-4">
+                <p><strong>Your new personal access token:</strong></p>
+                <pre class="bg-white border p-2 overflow-x-auto">{{ session('new_token') }}</pre>
```

Comment:
We have a handy dandy "copy to clipboard" js helper that should be added here for convenience